portal-api/src/v1/validation/fields/country.js

```diff
@@ -1,6 +1,15 @@
-const { findOneCountry } = require('../../controllers/countries.controller');
+const { getCountry } = require('../../controllers/countries.controller');
 
-module.exports.countryIsDisabled = (code) => {
-  const { data: country } = findOneCountry(code);
-  return country && country.disabled;
+/**
+ * Retrieves the disabled status of a country based on its country code.
+ * @param {string} code - The country code.
+ * @returns {boolean} - The disabled status of the country.
+ */
+const countryIsDisabled = async (code) => {
+  const { data } = await getCountry(code);
+  return Boolean(data?.disabled);
```

Comment:
I think this is good -- as long as we are sure we will always recieve a boolean as apposed to a string ie `Boolean("false")` would return `true`. Just raising this as this calls `external-api`

Reply:
With `Boolean()` return will always be logical boolean value as opposed to a `String`.